messages/4.12.0.txt

```diff
@@ -0,0 +1,14 @@
+SublimeLinter 4.12
+==================
+
+- Improved errors tooltip:
+  Now wraps long lines
+  Easily copy the error messages to you clipboar
```

Comment:
A d for the clipboard, and some flowers bc this was a contribution.

Reply:
Oh, sure, of course!